chanzuckerberg/single-cell-data-portal
# feat: retrieve only required attrs and dims [wip]

## Reason for Change

https://github.com/chanzuckerberg/single-cell-data-portal/issues/5214

## Changes

- passes in the `dims` and `attrs` as specified
- adds `compare` as a param to the query criteria objects for `WmgQueryCriteria` and `WmgQueryCriteriaV2`
- not entirely sure what the expected behavior should be for the `FmgQueryCriteria`, though maybe this doesn't matter that much since that will be deprecated hopefully soon

## Testing steps

- make sure existing tests pass
- TODO: write test coverage to make sure the `compare` param is being passed down correctly

## Notes for Reviewer
